include/inviwo/core/ports/imageport.h

```diff
@@ -181,8 +167,26 @@ struct PortTraits<BaseImageInport<0>> {
  *                    │                                  │                                  │
  *                    └──────────────────────────────────┴──────────────────────────────────┘
  * \endverbatim
+ *
+ * The ImageOutport records all the requested sizes from all it's connected inports. If the outport
+ * 'isHandlingResizeEvents' the port will resize its data to the largest of the requested sizes, And
+ * propagate that size an a new event upwards in the network. If it does not handle resize event it
+ * will not propagate any resize events nor resize it's data.
```

Comment:
Typos, change to
```c++
 * The ImageOutport records all the requested sizes from all its connected inports. If the outport
 * 'isHandlingResizeEvents', the port will resize its data to the largest the requested size and
 * propagate that size and a new event upwards in the network. If it does not handle resize events,
* it will not propagate any resize events nor resize its data.
```